Makefile

```diff
@@ -89,5 +89,7 @@ tmp/build/$(CLI_EXE_NAME)-darwin-amd64:
 help:
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-.PHONY: help
+check-license:
```

Comment:
Could you add a .PHONY and a help string for check-license please.
You can check the help string by running `make help`